plugin-docker/che-plugin-docker-client/src/test/java/org/eclipse/che/plugin/docker/client/DockerConnectorConfigurationTest.java

```diff
@@ -131,5 +143,98 @@ public void testDockerUriOnNonLinuxMissingProperties() throws Exception {
 
     }
 
+    /**
+     * Check that without property docker host ip from container is DEFAULT_LINUX_DOCKER_HOST_IP
+     */
+    @Test
+    public void testLinuxDefaultDockerHostWithoutBrige() throws Exception {
+        Map<String, String> env = new HashMap<>();
+        NetworkFinder networkFinder = Mockito.mock(NetworkFinder.class);
+        doReturn(Optional.empty()).when(networkFinder).getIPAddress(BRIDGE_LINUX_INTERFACE_NAME);
```

Comment:
I think you used incorrect API of mockito. This style is for spy objects, not mocks. For mocks use `when(networkFinder.getIPAddress(BRIDGE_LINUX_INTERFACE_NAME)).themReturn(...)`
From mockito documentation `Beware that {@link Mockito#when(Object)} is always recommended for stubbing because it is argument type-safe`